src/builder/create_sticker.rs

```diff
@@ -15,13 +15,14 @@ use crate::model::prelude::*;
 /// - [`PartialGuild::create_sticker`]
 /// - [`Guild::create_sticker`]
 /// - [`GuildId::create_sticker`]
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 #[must_use]
 pub struct CreateSticker<'a> {
     name: String,
     tags: String,
     description: String,
     file: CreateAttachment,
+    #[serde(skip)]
     audit_log_reason: Option<&'a str>,
 }
 
```

Comment:
@mkrasnitski so for `CreateSticker` the `#[serde(skip)]` does seem to work. Is this the best way to go? If so, then it seems like `DeserializeOwned` should be the way to go.